assets/stylesheets/application.scss

```diff
@@ -39,6 +38,13 @@ $govuk-link-underline-offset: .2em;
 @import "component/sub-navigation";
 @import "component/service-alert";
 
+@import "component/map/map";
+@import "component/header-bar";
+@import "component/list-filter/list-filter";
+
+@import "utilities/wrapper";
+@import "utilities/js-hidden";
+
```

Comment:
we were using some scss from the dl-land frontend. I've just pulled those files in and imported them here. 
as a future ticket. I think all the scss could be refined. but in the interest of not making this PR too big, that's for another ticket